build.ps1

```diff
@@ -8,6 +8,7 @@ param(
   [string[]] $SDKs = @("X64","X86","Arm64"),
   [string] $ProductVersion = "0.0.0",
   [string[]] $Test = @(),
+  [string] $StageArtifactsTo = "",
```

Comment:
How about `Stage` instead of `StageArtifactsTo`?  That makes the usage much simpler.

Reply:
Good idea, done!